src/elements/content-sidebar/activity-feed/version/Version.scss

```diff
@@ -6,7 +6,8 @@
         align-items: center;
         justify-content: space-between;
         width: 100%;
-        margin-top: 10px;
+        margin-top: $bdl-grid-unit * 4;
+        margin-bottom: $bdl-grid-unit * 4;
         padding: 9px;
```

Comment:
not part of your PR obvs, but this pixel value is a bit suspicious

Reply:
Yeah that's fair, can try to address it in a follow up